src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+import { deleteTodos } from '../../api/todos';
+
+interface Props {
+  activeTodosCount: number;
+  filter: Status;
+  onFilter: (status: Status) => void;
+  todos: Todo[];
+  onDeleteTodo: (id: number) => void;
+  onError: (error: string) => void;
+  onAddDeletingPostId: (id: number | null) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodosCount,
+  filter,
+  onFilter,
+  todos,
+  onDeleteTodo,
+  onError,
+  onAddDeletingPostId,
+}) => {
+  const todosIds = todos
+    .filter(({ completed }) => completed)
+    .map(({ id }) => id);
+
+  const handleDeleteTodos = () => {
+    onError('');
+
+    todosIds.forEach(id => {
+      onAddDeletingPostId(id);
+      deleteTodos(id)
+        .then(() => onDeleteTodo(id))
+        .catch(() => {
+          onError('Unable to delete a todo');
```

Comment:
please create enum for all error titles like 'Unable to delete a todo', 'Unable to load todos' etc,